examples/service-clients/azure-client/external-controller/package.json

```diff
@@ -11,13 +11,26 @@
 	},
 	"license": "MIT",
 	"author": "Microsoft and contributors",
-	"main": "dist/index.js",
-	"module": "lib/index.mjs",
-	"types": "dist/index.d.ts",
+	"type": "module",
+	"exports": {
+		".": {
+			"import": {
+				"types": "./lib/src/app.d.ts",
+				"default": "./lib/src/app.js"
+			},
+			"require": {
+				"types": "./dist/src/app.d.ts",
+				"default": "./dist/src/app.js"
+			}
+		}
+	},
+	"main": "dist/src/app.js",
+	"types": "dist/src/app.d.ts",
 	"scripts": {
 		"build": "fluid-build . --task build",
 		"build:compile": "fluid-build . --task compile",
-		"build:esnext": "tsc-multi --config ../../../../common/build/build-common/tsc-multi.esm.json",
+		"build:esnext": "tsc --project ./tsconfig.json",
+		"check:are-the-types-wrong": "attw --pack . --entrypoints .",
```

Comment:
'attw' is how I found the wrong entry points.  I debated whether it was worth it for examples.  I guess it is?